<launch>


	 <node name="pick_up_object_server" pkg="manipulation" type="pick_up_object_server.py" output="screen"> </node>
	 <node name="give_object_to_operator_server" pkg="manipulation" type="give_object_to_operator_server.py" output="screen"> </node>
	 <node name="receive_object_from_operator_server" pkg="manipulation" type="receive_object_from_operator_server.py" output="screen"> </node>
	 <node name="follow_server" pkg="manipulation" type="follow_server.py" output="screen"> </node>
	 
	 <node name="handle_grasp_pose_node" pkg="point_cloud_filtering" type="handle_grasp_pose" output="screen"> 
		<remap from="cloud_in" to="/hsrb/head_rgbd_sensor/depth_registered/rectified_points"/>
	 </node>

	 <node name="segment_object_node" pkg="point_cloud_filtering" type="segment_object" output="screen"> 
		<remap from="cloud_in" to="/hsrb/head_rgbd_sensor/depth_registered/rectified_points"/>
	 </node>


	  <!-- Load the collision map services -->
	  <include file="$(find manipulation)/launch/octomap_server.launch"> </include>
	  <include file="$(find manipulation)/launch/grasp_synthesis.launch"> </include>

</launch>
